tasks/shell_crmsh/cluster-setup-keys.yml

```diff
@@ -0,0 +1,35 @@
+# SPDX-License-Identifier: MIT
+---
+- name: Get corosync authkey
+  ansible.builtin.include_tasks: ../presharedkey.yml
+  vars:
+    preshared_key_label: corosync authkey
+    preshared_key_src: "{{ ha_cluster_corosync_key_src }}"
+    preshared_key_dest: /etc/corosync/authkey
+    preshared_key_length: 256
+
+- name: Distribute corosync authkey
+  ansible.builtin.copy:
+    content: "{{ __ha_cluster_some_preshared_key | b64decode }}"
+    dest: /etc/corosync/authkey
+    owner: root
+    group: root
+    mode: '0400'
+  register: __ha_cluster_distribute_corosync_authkey
```

Comment:
not sure if `content` is marked as a `no_log` parameter in the `copy` module, but if you see it being logged with `ansible-playbook -vv`, you should mark this task with `no_log: true` - same with `Distribute pacemaker authkey` below

Reply:
Fixed, but pcs code also does not have it. Do you want to include it as well?

Reply:
It seems to be `no_log: true`: https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/copy.py#L537

Reply:
@tomjelinek confirms `no_log: true` is default for `copy` module, so is this change really needed?

Reply:
> It seems to be `no_log: true`: https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/copy.py#L537

Ok - as long as no cleartext passwords/keys are logged with `-vv`, you don't have to do anything